source/CardGames.py

```diff
@@ -266,23 +211,135 @@ def highestCard(cardList):
       highestCard = card
   return highestCard
 
-def FugitiveFirst():
-  escape_card, HighRangeDeck, MidRangeDeck, LowRangeDeck, starting_cards = split_card()
-  fugitive_deck = starting_cards
-  fugitive_deck.append(escape_card)
-  for x in range(3):
-    fugitive_deck.append(LowRangeDeck.pop())
-                    
-  for x in range(2):
-    fugitive_deck.append(MidRangeDeck.pop())
+  
+
+
+class GameSetup:
+    def __init__(self):
+        #Initialize the objects that need to be initialized
+        self.escape_card, self.HighRangeDeck, self.MidRangeDeck, self.LowRangeDeck, self.starting_cards = split_card()
+        self.cards_in_play = []
+        self.fugitive = Player("", "Fugitive", 0)
```

Comment:
Looks great! You can probably just remove the 0 parameter in the player initialization since that's money and we're probably going to remove the money anyways.

Reply:
Thanks Sean! I went ahead and made those changes.